contribs/gmf/src/datasource/OGC.js

```diff
@@ -4,7 +4,6 @@ import ngeoDatasourceOGC from 'ngeo/datasource/OGC.js';
 /**
  * The options required to create a `gmf.datasource.OGC`.
  * @typedef {Object} OGCOptions
- * @extends import('ngeo/datasource/OGC.js').OGCOptions
```

Comment:
Hum...  I don't understand how this is supposed to affect the type checking yet, but I won't prevent you from merge just because of this.  I'll figure it out later.

Reply:
One good example of using extends is the following:
```
/**
 * @extends A<number>
 */
class B extends A {...}
```
To the value of the template.
In this case it's incoherent, then I don't know what it can mean...